### ISSUE TITLE
PhantomJS not running on OSX Sierra?

### ISSUE THREAD

**Issue by thescientist13** [thegreenhouseio/angular-webapp-seed#47]
In trying to debug #46 locally, it seems PhantomJS isn't starting on my machine
```
webpack: bundle is now VALID.
10 12 2016 20:46:28.441:INFO [karma]: Karma v1.2.0 server started at http://localhost:9876/
10 12 2016 20:46:28.441:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
10 12 2016 20:46:28.456:INFO [launcher]: Starting browser PhantomJS
10 12 2016 20:46:28.682:ERROR [launcher]: Cannot start PhantomJS

10 12 2016 20:46:28.687:INFO [launcher]: Trying to start PhantomJS again (1/2).
[default] Checking started in a separate process...
10 12 2016 20:46:28.898:ERROR [launcher]: Cannot start PhantomJS

10 12 2016 20:46:28.899:INFO [launcher]: Trying to start PhantomJS again (2/2).
10 12 2016 20:46:29.120:ERROR [launcher]: Cannot start PhantomJS

10 12 2016 20:46:29.121:ERROR [launcher]: PhantomJS failed 2 times (cannot start). Giving up.
[default] Ok, 1.383 sec.
```

**Comment by thescientist13** [comment]
kind of blocks debugging of https://github.com/thegreenhouseio/angular2-webpack-seed/issues/46  : /

**Comment by thescientist13** [comment]
If i could fix #45 , then maybe I could get around this issue.
